src/pages/ProfileForm/CreateProfileForm/index.tsx

```diff
@@ -0,0 +1,153 @@
+  };
+
+  const {
+    register,
+    control,
+    reset,
+    trigger,
+    getValues,
+    formState: { errors: formErrors },
+  } = useForm<ValidationSchemaCPF>({
+    resolver: zodResolver(validationSchemaCPF),
+    defaultValues,
+  });
+
+  const { fields, append, remove } = useFieldArray({
+    name: 'financialInstitutions',
+    control,
+  });
+
+  const onAppendFinancialInstitutions = (): void =>
+    append(emptyAddFinancialInstitution);
+
+  // NOTE: This function is used for submitting the multipart/formData
+  const onSubmitButtonAction = async (): Promise<void> => {
+    const passesValidation = await trigger();
+    if (passesValidation) {
+      const preFormattedData = getValues();
+      // POST formData
+      // TODO: Will be used for debugging after clicking 'Submit'
+      // eslint-disable-next-line no-console, @typescript-eslint/no-unused-vars
+      const response = await submitUserProfileFi(
+        auth,
+        preFormattedData,
+        `${preFormattedData.firstName} ${preFormattedData.lastName}`,
+      );
+    } else {
+      scrollToElement(formErrorHeaderId);
+    }
+  };
+
+  const onClearform = (): void => {
+    reset();
+    scrollToElement('firstName');
+  };
+
+  return (
+    <FormWrapper>
+      <div id='update-financial-profile'>
+        <FormHeaderWrapper>
+          <CrumbTrail>
+            <Link href='/'>Platform home</Link>
+          </CrumbTrail>
+          <Step1FormHeader crumbTrailMarginTop />
+        </FormHeaderWrapper>
+        <Step1FormInfoHeader />
+        <FormErrorHeader errors={formErrors} id={formErrorHeaderId} />
+
+        <Step1FormInfoFieldGroup formErrors={formErrors} register={register} />
+        <SectionIntro heading='Provide your financial institution details'>
+          Provide the name, LEI, and RSSD ID of the financial institution for
+          which you are authorized to file. If you have an RSSD ID, you must
+          provide it. If you are authorized to file for an additional financial
+          institution, click “Add a financial institution”.
+        </SectionIntro>
+        {fields.map((field, index) => {
+          const onRemoveThisInstitution = (): void => remove(index);
+          return (
+            <div className='flex flex-col' key={`${field.id}`}>
+              {index !== 0 && (
+                <LinkButton
+                  className='ml-auto'
+                  icon='minus'
+                  onClick={onRemoveThisInstitution}
+                >
+                  Remove this financial institution
+                </LinkButton>
+              )}
+              <AddFinancialInstitution
+                index={index}
+                register={register}
+                formErrors={formErrors}
+              />
+            </div>
+          );
+        })}
+        <div className='mb-[3.75rem]'>
+          <LinkButton onClick={onAppendFinancialInstitutions} icon='plus'>
+            Add a financial institution
+          </LinkButton>
```

Comment:
Also used in the `Update your user profile` page. Small element but may want to split into custom component for consistency.